go.mod

```diff
@@ -7,6 +7,11 @@ retract (
 	[v1.6.0, v1.6.1] // Contains data race bug in background connection establishment.
 )
 
+//gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c introduced through github.com/stretchr/testify@v1.6.1 are
+// vulnerable to Denial of Service (DoS) via the Unmarshal function, which causes the program to crash when attempting
+// to deserialize invalid input. https://www.cve.org/CVERecord?id=CVE-2022-28948
+replace gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c => gopkg.in/yaml.v3 v3.0.1
```

Comment:
@matthewdale as you've noted, updating to a version of testify that resolves CVE-2022-28948 is not something we can do without breaking support for Go v10. Using [replace](https://go.dev/ref/mod#:~:text=A%20replace%20directive%20replaces%20the,a%20platform%2Dspecific%20file%20path.) updates the vulnerable yaml.v3 version to 3.0.1, which removes it from go.sum. However, the testify package still points to `v3.0.0-20200313102051-9f266ea9e77c` in the dependency graph:

```
github.com/stretchr/testify@v1.6.1 gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c
```